source/hic_hal/atmel/sam3u2c/usb_config.c

```diff
@@ -28,8 +30,12 @@
 
 //   <o0.0> High-speed
 //     <i> Enable high-speed functionality (if device supports it)
-#define USBD_HS_ENABLE              1
-
+#define USBD_HS_ENABLE              0
```

Comment:
I had to disable HS functionality. Windows is showing invalid descriptor message when it is enabled. @c1728p9 any idea what may be wrong?

Reply:
This patch makes some changes to the full speed configuration descriptor ```USBD_ConfigDescriptor``` which weren't made to the high speed configuration descriptor ```USBD_ConfigDescriptor_HS```. I left comments below as well in the full speed descriptor asking about this.